Formula/nmap.rb

```diff
@@ -21,7 +21,12 @@ class Nmap < Formula
     sha256 x86_64_linux:   "5ceab0e20f0aba5059b7ba612876413c799d0a933dfb46c5bf078b432d01c7dd"
   end
 
+  depends_on "liblinear"
+  depends_on "libssh2"
+  # Check supported Lua version at https://github.com/nmap/nmap/tree/master/liblua.
```

Comment:
Could we have a test for this in the same way we have a test for teleport webassets?

Reply:
Added one inside the `install` method, since it requires parsing the `configure` script. The alternative would be to parse Git logs, but who knows in what format those will appear.